csharp/src/Microsoft.ML.OnnxRuntime/NativeOnnxTensorMemory.cs

```diff
@@ -17,31 +17,29 @@ internal class NativeOnnxTensorMemory<T> : MemoryManager<T>
         private bool _disposed;
         private int _referenceCount;
         private IntPtr _onnxValueHandle;
-        private IntPtr _dataBufferHandle;
+        private IntPtr _dataBufferPointer;
+        private string[] _dataBufferAsString;
```

Comment:
may be add some comment explaining what it stores

Reply:
done